.github/workflows/daphneci.yml

```diff
@@ -39,8 +39,35 @@ jobs:
     steps:
       - name: Checking out
         uses: actions/checkout@v3
+
+      - name: Generate HPKE Signing Key
+        id: hpke_signing_key
+        run: |
+          key=$(openssl ecparam -name prime256v1 -genkey -noout -out -)
+          {
+            echo "hpke_signing_key<<EEEEEOF"
+            echo "$key"
+            echo "EEEEEOF"
+          } >> "$GITHUB_OUTPUT"
+
+      - name: Generate HPKE Signing Certificate
+        id: hpke_signing_cert
+        run: |
+          echo "${{ steps.hpke_signing_key.outputs.hpke_signing_key }}" > private-key
+          cert=$(openssl req -key private-key \
+                      -new -x509 -days 365 -out - \
+                      -subj '/C=PT/L=Braga/O=Cloudflare Lda/CN=cloudflare.com')
```

Comment:
CN=cloudflare.com is probably not a great DN to use. In practice, this would be a test Leader, correct?